src/components/shared/FormInfo.js

```diff
@@ -12,135 +13,139 @@ const StyledGroup = styled.div`
     }
 `;
 
-class FormInfo extends Component {
-    constructor(props) {
-        super(props);
+const FormInfo = (props) => {
+    let elemInputs = [];
+    let saveInputs = [];
+
+    const initialState = {
+        id: '',
+        title: null,
+        date: null,
+        url: null,
+        overview: null,
+        time: null
+    };
 
-        this.initialState = {
-            id: '',
-            title: null,
-            date: null,
-            url: null,
-            overview: null,
-            time: null
-        };
+    const [genreState] = useState(['Select Genre', 'Horror', 'Action', 'Comedy']);
+    const [valueState, setValue] = useState(genreState[0]);
+    const [dataForm, setData] = useState(initialState);
 
-        this.newState = null;
+    const handleChange = useCallback((e) => {
+        setValue(e.target.value);
+        }, []
+    );
 
-        this.elemInputs = [];
-        this.saveInputs = [];
+    const getInputs = () => {
+        if (props.showEditPage) {
+            elemInputs = document.querySelectorAll('.Edit input');
 
-        this.state = {
-            value: 'Select Genre',
-            genres: ['Select Genre', 'Horror', 'Action', 'Comedy'],
-            dataForm: Object.assign({}, this.initialState)
+        } else if (props.showAddPage) {
+            elemInputs = document.querySelectorAll('.Add input');
         }
-    }
-
-    handleChange = (e) => {
-        this.setState({ value: e.target.value })
-    }
-
-    handleReset = () => {
-        if (this.props.showEditPage) {
-            this.elemInputs = document.querySelectorAll('.Edit input')
+    };
 
-        } else if (this.props.showAddPage) {
-            this.elemInputs = document.querySelectorAll('.Add input')
-        }
+    const handleReset = useCallback(() => {
+        getInputs();
 
-        if (this.elemInputs.length) {
-            this.elemInputs.forEach(input => {
-                this.saveInputs.push(input.value);
+        if (elemInputs.length) {
+            elemInputs.forEach(input => {
+                saveInputs.push(input.value);
                 input.value = '';
             });
         }
 
-        this.setState({ value: 'Select Genre' });
-    }
+        setValue(genreState[0]);
+    });
```

Comment:
Using `useCallback` without the second argument - an array of depth - is completely useless.
Please, read more https://reactjs.org/docs/hooks-reference.html#usecallback